WordPress/src/main/java/org/wordpress/android/ui/sitecreation/domains/SiteCreationDomainsViewModel.kt

```diff
@@ -193,25 +181,21 @@ class SiteCreationDomainsViewModel @Inject constructor(
                 .toList().flatten()
 
             // We inform the user when the search query contains non-alphanumeric characters
-            val emptyListMessage = if (event.isError && event.error.type == SuggestDomainErrorType.INVALID_QUERY) {
-                UiStringRes(R.string.new_site_creation_empty_domain_list_message_invalid_query)
-            } else {
-                UiStringRes(R.string.new_site_creation_empty_domain_list_message)
-            }
+            val emptyListMessage = UiStringRes(
+                if (event.isError && event.error.type == SuggestDomainErrorType.INVALID_QUERY)
+                    R.string.new_site_creation_empty_domain_list_message_invalid_query
+                else R.string.new_site_creation_empty_domain_list_message
+            )
```

Comment:
Idk, the reason I'm formatting if-else's like this is because I actually miss the ternary operator (`tru ? dis : dat`) in Kotlin, probably the most significant thing it lacks 😅 . Tbh I prefer if else without curly braces if it's on one line (which is the case by my reasoning 😅 ), and this is (quite) widely used in the codebase.

Why they recommend it in the official style guide it's not clear to me 🤷